docs/MERGIFY.md

```
@@ -0,0 +1,163 @@
+
+Common reasons:
+- ❌ CI check still running or failed
+- ❌ Merge conflicts present
+- ❌ PR is a draft
+- ❌ Title doesn't match expected pattern
+- ❌ Major version update needs approval
+
+### Forcing a recheck
+
+Comment on the PR:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)